src/features/useRssiDevice.ts

```diff
@@ -32,6 +33,8 @@ export default () => {
                     getState().app.rssi.scanRepeat
                 );
             });
+            dispatch(setIsPaused(false));
```

Comment:
Looks good, but what if setting "initial state" after device is selected?
Whenever "initial state" is updated you don't need to think about new properties that should be updated

Reply:
But initial state will replace the side panel settings user might have made

Reply:
yep, ideally they should be separated. like 
`{
 initialState: {...}
 filters: {...}
`